### DISCUSSION TITLE
Configuration.yaml mancante

### DISCUSSION THREAD

**Discussion by DokMing** [indomus/forum#606] [HUB: Home Assistant]
Salve, mi presento: mi chiamo Cristiano, sono di Ferrara provincia e sono un niubbo su HA. Tra i vari hobby ho da poco la domotica e ho trovato questo software che come hub viene presentato molto bene e per qual poco che posso capire in merito e' proprio cio' che fa' al caso mio.

Ora passo alla domanda principale.

Come e' possibile che io non abbia il file configuration.yaml ed in caso lo creassi al riavvio o cmq dopo poco non lo trovo piu'... viene cancellato.

L'installazione e' stata effettuata su un rasp3 con docker percio' niente SO su cui effettuare l'installazione; per spiegarmi meglio allego la schermata che mi compare appena mi loggo in ssh.
![Immagine 2021-08-11 201938](https://user-images.githubusercontent.com/88689078/129082390-255dc215-5e20-40cd-8342-c07665ef1ef3.png)

Spero che qualcuno possa aiutarmi o almeno spiegarmi dove vengono salvati tutti i dati di sistema e cosa devo fare per rimmediare.

Grazie mille a chiunque possa darmi una mano.

**Comment by snakuzzo** [comment]
Hai installato HassOS. Il file `configuration.yaml` si trova sotto `/config` e te lo trovi già lì al primo avvio

**Reply by DokMing** [reply]
Instanto grazie per la risposta e ciao. Questo che dici lo pensavo anche io ma non e' cosi' in /config l'unica cosa che trovo sono 2 cartelle .storage e .cloud
Il bello, per altro e' che anche se vado a crearlo con un touch, nano, vi, o altro...poi viene cancellato o comunque non lo trovo piu'. Facendo anche un find o un locate dalla root in giu' non risulta essercene altri.
La cosa mi sembra strana!

**Reply by snakuzzo** [reply]
Guarda che quei file te li ritrovi appena avvii la prima volta HassOS.
Non so cosa tu possa aver fatto...ci vorrebbe una sfera magica :D
Eppure ti bastava flashare l'immagine e farlo partire

**Reply by DokMing** [reply]
snakuzzo ok tutto ma non sono proprio cosi' assente nel mondo informatico;-) 
Cmq ora sto' riprovando con una installazione tramite balena e come url dell'image
https://github.com/home-assistant/operating-system/releases/download/6.2/haos_rpi3-64-6.2.img.xz

**Reply by DokMing** [reply]
Scusa snakuzzo preciso perche' non vorrei essere stato saccente o villano. Sono niubbo su Hassio ma non di iformatica.
e cmq ripeto grazie della prima risposta e visto la tua disponibilita' sono sicuro anche delle prossime. Grazie mille.

**Reply by DokMing** [reply]
riciao. Rifatta l'installazione e di seguito lo screen
![Immagine 2021-08-12 120717](https://user-images.githubusercontent.com/88689078/129179310-48fe684f-16ce-4edd-8a9c-f38832fc92c1.png)
Mi viene da dire, per ora, che mi sono spaccato la testa su una installazione che era fallata dall'origine.
